.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# format changes: no-var, prefer-const
+a828ce6ab7d06077231bfe6f7827f9c9cf05f4ef
```

Comment:
shouldn't you add it only after merging to master?

Reply:
(@dannyzaken I added numbers):

> 1. what is this revision? 
> 2. Shouldn't you add it only after merging to master?

1. I'm not sure I understand the first question. if you asked about the file `.git-blame-ignore-revs` it is a file that allows us to do changes that will not be reflected in `git blame` option (in Github as well). [here](https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view) is the link for more information.
2. I did it in a separate commit, I will do it in a separate PR (after merging this PR).